edx_proctoring/api.py

```diff
@@ -193,6 +191,34 @@ def remove_review_policy(exam_id):
     exam_review_policy.delete()
 
 
+@receiver(post_save, sender=ProctoredExamReviewPolicy)
+@receiver(post_save, sender=ProctoredExam)
+def _save_exam_on_backend(sender, instance, **kwargs):  # pylint: disable=unused-argument
+    """
+    Save the exam to the backend provider when our model changes.
+    It also combines the review policy into the exam when saving to the backend
+    """
+    if sender == ProctoredExam:
+        exam_obj = instance
+        review_policy = ProctoredExamReviewPolicy.get_review_policy_for_exam(instance.id)
+    else:
+        exam_obj = instance.proctored_exam
+        review_policy = instance
+    if exam_obj.is_proctored:
+        exam = ProctoredExamSerializer(exam_obj).data
+        if review_policy:
+            exam['rule_summary'] = review_policy.review_policy
+            # When the rules are defined as boolean options,
+            # save them here
+            if hasattr(review_policy, 'rules'):  # pragma: no cover
+                exam['rules'] = review_policy.rules  # pragma: no cover
```

Comment:
Why do we think these lines can't be tested?

Reply:
@nedbat because @MichaelRoytman hasn't finished his PR, which will create the rules field on the review policy model. I assume that he will remove the pragma comment and the hasattr check in his PR.